lib/pretty_text.rb

```diff
@@ -225,6 +225,10 @@ def self.markdown(text, opts = {})
 
       if opts[:user_id]
         buffer << "__optInput.userId = #{opts[:user_id].to_i};\n"
+
+        # NOTE: If using this for server-side cooking, you will want to do
+        # user.reload before accessing data on this parsed User, otherwise
+        # AR relations will not be loaded.
```

Comment:
If this has already been converted to JSON, will we still be able to call `reload` on it?

Reply:
When you send it back through to a `PrettyText::Helper` through a shim, it gets converted to a Hash, then you can load an AR model from that. Just wanted to leave this here so noone made the same mistake I did :)

Reply:
ooo in this case I think what we want to say is that the variable here represents a hash not an ActiveRecord model? Because the suggestion of calling `reload` only works if we're dealing with an ActiveRecord model which we're not.

Reply:
Yes I see what you are saying, I will say that and add a caveat to say if you use the hash to make an AR model _then_ you need to reload :+1: